specification/metrics/billedcost.md

```diff
@@ -1,8 +1,8 @@
 # Billed Cost
 
-The Billed Cost represents a charge inclusive of negotiated discounts that a consumer would be charged for each billing period. Billed Cost does not amortize upfront charges (one-time or recurring). The currency for the value specified in Billed Cost can be found in the [Billing Currency](#billingcurrency) column. Billed Cost is often used to perform FinOps capabilities that require cash-basis accounting such as cost allocation, budgeting, and invoice reconciliation.
+The Billed Cost represents a charge inclusive of the impacts of volume/tier-based rates, negotiated discounts, and commitment-based discounts while not amortizing upfront charges (one-time or recurring). This cost is denominated in the [Billing Currency](#billingcurrency). The Billed Cost is used to perform FinOps capabilities that require cash-basis accounting such as cost allocation, budgeting, and invoice reconciliation.
```

Comment:
```suggestion
The Billed Cost represents a charge inclusive of all reduced rates and discounts while not amortizing upfront charges (one-time or recurring). This cost is denominated in the [Billing Currency](#billingcurrency). The Billed Cost is used to perform FinOps capabilities that require cash-basis accounting such as cost allocation, budgeting, and invoice reconciliation.
```

Reply:
Given the relationship between List -> Negotiated -> Billed, I think being more explicit is better than grouping into 'all'

Reply:
B looks good

Reply:
Discussed and resolved at the meeting. Opted for: _A charge serving as the basis for invoicing, inclusive of all reduced rates and discounts while excluding the amortization of upfront charges (one-time or recurring)._